### ISSUE TITLE
Use geo as default CPT for remote earth_relief files

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt#810]
The GMT default CPT is (still) rainbow, both loved and hated by mappers.  Yet, if you are making a map of a DEM grid then it is helpful that there is a sharp contrast at the zero contour.  For instance, to demonstrate how simple it is to make a global map of Earth at a small scale, I can do this:

`gmt grdimage @earth_relief_10m -JH0/10c -I+d -png earth
`

![earth](https://user-images.githubusercontent.com/26473567/58659621-21444f00-82bf-11e9-92cc-e7e36632893b.png)

But this looks crappy: The CPT has no natural zero and the green of the land bleeds into the ocean until there is a noticeable change to blue at some depth.  Also, almost all of the land is green.  In other words, this plot is simply useless.  In contrast, if I use **-C** to select another CPT such as geo, I get quite a different map:

`gmt grdimage @earth_relief_10m -JH0/10c -I+d -Cgeo -png earth2
`

![earth2](https://user-images.githubusercontent.com/26473567/58659628-2903f380-82bf-11e9-90c4-a111edf2c773.png)

My proposal is simply that when no CPT is requested we select geo as the default CPT when earth_relief remote files are involved; otherwise we stick to rainbow.

**Comment by seisman** [comment]
I have a concern here. If I already have earth_relief data in my `~/.gmt/servers` or current directory, I can use it  without the @ prefix.

If your proposal is implemented, the following two similar commands will generate totally different images, due to the different default CPT used, right?
```
gmt grdimage @earth_relief_10m.grd -JH0/10c -I+d -png earth2
gmt grdimage earth_relief_10m.grd -JH0/10c -I+d -png earth2
```

**Comment by PaulWessel** [comment]
We could do it for files named earth_relief_xxxr instead. But _not_ using @ for these files (even if you have a copy) is not ideal since if there is a better version you would not get a replacement.  So we either

1. Require that @ is used (it goes locally first anyway) to set geo.cpt
2.Ignore @ and instead look for earth_relief_xxxr files.

**Comment by seisman** [comment]
OK, then I prefer the first way, using @ as a magic symbol.

**Comment by joa-quim** [comment]
Well, sorry, but I don't see why '@' should determine the cpt to use. The rational is that the geo.cpt is better for earth_xxx files, so let's use it it based on name, not on access type.

**Comment by PaulWessel** [comment]
Sure, I meant @earth_relief*, not just @.  @earth_ages_02m may not work well with geo.

**Comment by joa-quim** [comment]
Sorry, I also meant earth_relief (with or without the @)

**Comment by PaulWessel** [comment]
I hesitate on that because it is very conceivable that a user could use that name for a file of theirs and be bewildered.  We have no monopoly on earth_relief.  However, I admit it is unlikely, and we can document it very clearly.

**Comment by PaulWessel** [comment]
Oh, one more response to @seisman : You don't really want to make an autoscaled earth relief map with the rainbow CPT since it will be what you see above.  Surely that is not a desired map?

**Comment by joa-quim** [comment]
No one should be bewildered if a default behavior is not what he expects.

**Comment by PaulWessel** [comment]
Surely you are joking, Dr. Luis.

**Comment by seisman** [comment]
Closed by #815
